uint8 Empty
---
uint8 Data